src/elementary/LevelContinuum.hpp

```diff
@@ -0,0 +1,80 @@
+#ifndef NJOY_ELEMENTARY_LEVELCONTINUUM
+#define NJOY_ELEMENTARY_LEVELCONTINUUM
+
+// system includes
+#include <regex>
+#include <stdexcept>
+#include <string>
+
+// other includes
+#include "elementary/LevelNumber.hpp"
+
+namespace njoy {
+namespace elementary {
+
+  /**
+   *  @class
+   *  @brief The nuclear level continuum
+   */
+  class LevelContinuum {

```

Comment:
This name seems a little odd to me. Would it make sense to make a `level` namespace?

Reply:
I'm actually considering of putting this in the LevelNumber class (and renaming LevelNumber to Level). The continuum range of levels is currently unused so I don't really need the second level number anyway. Once we do need it, we can always redesign things.

Reply:
I don't want to spend a lot of time doing redesign. We are kind of at a time crunch right now.

Reply:
I'm talking about a redesign later, when we need a continuum between given levels